go.mod

```diff
@@ -44,7 +46,7 @@ require (
 	github.com/go-logr/stdr v1.2.2 // indirect
 	github.com/gofrs/flock v0.8.1 // indirect
 	github.com/gogo/googleapis v1.4.1 // indirect
-	github.com/gogo/protobuf v1.3.2 // indirect
+	github.com/gogo/protobuf v1.3.2 // indirect; indirec1
```

Comment:
```suggestion
	github.com/gogo/protobuf v1.3.2 // indirect
```

Reply:
I can try `go mod tidy` again on this, this wasn't my doing, though.

Reply:
Ok manually fixed and go mod left it be. 🤷 
Don't know what was up with that.